dataservice/api/README.md

```diff
@@ -1,4 +1,58 @@
-Primary API for interacting with the Kid's First Data Model.
+<center>
+![Kids First](/logo)
+</center>
+
+Welcome to the developer documentation for the Kids First Dataservice API!
+
+The dataservice is the primary hub for retrieval and creation of Kids First
+data. It exposes functionality using a familiar REST interface over HTTP
+requests.
+
+# Connect to the API
+
+The API is currently only available internally inside the Kids First cloud
+environments.
+
+## curl
+
+Curl is the quickest way to communicate with the api from a unix shell
+
+```bash
+curl -H "Content-Type: application/json" http://kf-api-dataservice-qa.kids-first.io/status
+```
+
+To give input to the API:
+```bash
+curl -H "Content-Type: application/json" http://kf-api-dataservice-qa.kids-first.io/status -d '{ "external_id": "my identifier" }'
```

Comment:
Since we can't post to /status how about something a little more realistic like :

```bash
curl -X POST -H "Content-Type: application/json" http://kf-api-dataservice-qa.kids-first.io/studies -d '{ "external_id": "my identifier" }'
```